integration-test-groups/foundation/scheduler/src/main/java/org/apache/camel/quarkus/component/scheduler/it/SchedulerResource.java

```diff
@@ -33,11 +31,15 @@
 public class SchedulerResource {
 
     @Inject
-    @Named("myCounter")
-    AtomicInteger myCounter;
+    @Named("withDelayCounter")
+    AtomicInteger withDelayCounter;
 
-    @Named("shoppingList")
-    CopyOnWriteArrayList<Integer> shoppingList;
+    @Inject
+    @Named("useFixedDelayCounter")
+    AtomicInteger useFixedDelayCounter;
+
+    @Named("withDelayRepeat")
+    AtomicInteger withDelayRepeat;
```

Comment:
Maybe `withDelayRepeatCounter` would align with other names ? What do you think ?

Reply:
Yes sounds good!